src/Xamarin.Android.Build.Tasks/Tasks/JavaSourceUtils.cs

```diff
@@ -139,11 +140,15 @@ string CreateResponseFile ()
 			}
 			if (!string.IsNullOrEmpty (JavadocUrlPrefix)) {
 				AppendArg (response, "--doc-url-prefix");
-				AppendArg (response, Path.GetFullPath (JavadocUrlPrefix));
+				AppendArg (response, JavadocUrlPrefix);
```

Comment:
So several of these weren't even files? And we were doing `GetFullPath()`?

Reply:
Yeah, I think this may have just been an oversight initially.